migration/src/model/table.rs

```diff
@@ -29,3 +29,17 @@ pub enum UTXOs {
     SlotUpdated,
     CreatedAt,
 }
+
+#[derive(Copy, Clone, Iden)]
+pub enum TokenOwnership {
```

Comment:
I'd consider calling this `TokenOwners` to be consistent with the other tables (pluralized)

Reply:
Makes sense